tests/app/auth/login/login.component.spec.ts

```diff
@@ -36,10 +36,45 @@ describe('LoginComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should start login process', () => {
-    component.startLoginProcess();
+  it('should call startLoginProcess with false in startPublicOnlyLoginProcess', () => {
+    spyOn(component, 'startLoginProcess');
 
-    expect(logger.info).toHaveBeenCalled();
-    expect(authService.startOAuthProcess).toHaveBeenCalled();
+    component.startPublicOnlyLoginProcess();
+
+    expect(component.startLoginProcess).toHaveBeenCalledWith(false);
+  });
+
+  it('should call startLoginProcess with true in startIncludePrivateLoginProcess', () => {
+    spyOn(component, 'startLoginProcess');
+
+    component.startIncludePrivateLoginProcess();
+
+    expect(component.startLoginProcess).toHaveBeenCalledWith(true);
+  });
+
+  it('should call authService.startOAuthProcess on startLoginProcess', () => {
+    const hasPrivateConsent = false;
+
+    component.startLoginProcess(hasPrivateConsent);
+
+    expect(authServiceSpy.startOAuthProcess).toHaveBeenCalledWith(hasPrivateConsent);
+    expect(loggingServiceSpy.info).toHaveBeenCalledWith('LoginComponent: Beginning login process');
+  });
+
+  it('should call authService.changeAuthState and errorHandlingService.handleError', () => {
```

Comment:
Maybe can change the spec name to `should call error handling methods when error is thrown` or something - just to make clear that you're testing the error handling.